src/RewardsDistributor.sol

```diff
@@ -17,6 +18,9 @@ contract RewardsDistributor is IRewardDistributor {
     address public payoutToken;
     string public name;
 
+    uint256 public precision;
```

Comment:
small optimization: you could save one storage slot access per payout per rd by changing `precision` to uint128 and moving definition after poolId

Reply:
```
Error (7407): Type uint256 is not implicitly convertible to expected type uint128.
  --> src/RewardsDistributor.sol:38:21:
   |
38 |         precision = 10 ** IERC20(payoutToken_).decimals();
```